onboarding.md

```diff
@@ -0,0 +1,36 @@
+# Developers Onboarding
+
+## Welcome in to our developers hub 
+
+We look forward to working together!
+
+## Start Here 
+
+**What is Cambiatus?**
+
+Open online platform that enables communities/groups with environmental and social objectives to create and implement their own community currencies. 
+
+At Cambiatus we make the blockchain technology available and user friendly, so these communities can take full advantage of it and solve their environmental or social problems. Cambiatus is also becoming a tool to make DAC (decentralized autonomous communities) creation easy and fun. 
+
+**What is DAC?**
+
+We have put together a document called the [Path to Knowledge](https://docs.google.com/document/d/1cKmTSR9VpPkYyD-uhpnnAuF4JITGWSu6X5BP9le1yMw/edit) in which we offer some sources of information which you can follow to learn about all the above topics. 
+
+**Now what really means Cambiatus?**
+
+In Spanish “Cambia” is the word used to express “Change”.  Cambiatus refers to a change in our mindset, a change in the way we understand money, organizations and collaboration. 
+
+When people start using Cambiatus methodologies and tools, something changes inside them, they get it! 
+
+## Development Environments
+
+Now that you have your Cambiatus mindset properly setup is time to have some tech fun!
+
+Here are our development environments documentation to help you dive in and setup:
+
+- [Backend](backend.md)
```

Comment:
we can link all of those to their repos, and link back to wiki for more specific concepts and cambiatus developer only. Because we can have contributors and devs that integrate their apps with us

Reply:
Gotcha!